QuantLib/ql/models/model.cpp

```diff
@@ -35,23 +38,24 @@ namespace QuantLib {
     CalibratedModel::CalibratedModel(Size nArguments)
     : arguments_(nArguments),
       constraint_(new PrivateConstraint(arguments_)),
-      shortRateEndCriteria_(EndCriteria::None) {}
+      endCriteria_(EndCriteria::None) {}
```

Comment:
argh... hard to live with a nonsensical shortRateEndCriteria_, especially when there is a dedicated inspector such as
EndCriteria::Type endCriteria() const;

If we document the breaking changes to CalibrationHelpers and CalibratedModel this could go with them, otherwise I agree we could go back to shortRateEndCriteria_ :-(